docs/src/examples.md

```diff
@@ -1,17 +0,0 @@
-# Examples
-
-See the [examples](https://github.com/googleforgames/quilkin/tree/{{GITHUB_REF_NAME}}/examples) folder on Github for configuration and 
```

Comment:
I couldn't find this content anywhere - maybe add it back in under **Deployment** as the last item?

Or maybe it's better with a table that actually lists all the examples?

Reply:
I've now included this information in deployment/examples